main/src/library/File.flix

```diff
@@ -840,4 +840,100 @@ namespace File {
                 Err(getMessage(ex))
         }
 
+    ///
+    /// Returns a list naming the files and directories in the directory `f`.
```

Comment:
We should indicate in the documentation whether the returned strings are the full path or just the file names.